openassessment/management/tests/test_upload_oa_data.py

```diff
@@ -69,4 +67,4 @@ def test_upload(self):
 
         # Expect that we generated a URL for the bucket
         url = cmd.history[0]['url']
-        self.assertIn("https://{}".format(self.BUCKET_NAME), url)
+        self.assertIn("ttps://s3.amazonaws.com/{}".format(self.BUCKET_NAME), url)
```

Comment:
Terrible copy-paste job by the dev who put this here, what a bum that guy must be (thanks, will fix)